<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameImageFileClass</class>
 <widget class="QDialog" name="RenameImageFileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RenameImageFile</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>RenameImageFile</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Source Folder:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dest Folder:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sourceFolder">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_destFolder">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sourceFolder">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browser...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_destFolder">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browser...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_selfFolder">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Self Folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_execute">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_destFileName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>100</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Source File Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sourceFileName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>100</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Dest File Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>FileType:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_fileType">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
